app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php

```diff
@@ -178,8 +176,7 @@ public function form(Form $form): Form
                                     ->visible(fn (Get $get) => $get('is_custom_link_url_enabled'))
                                     ->columnSpanFull(),
                             ]),
-                    ])
-                    ->visible(fn () => AddBrandedWebsitesToThemeSettingsFeature::active()),
+                    ]),
```

Comment:
This doesn't seem to be the Feature Flag mentioned in the title of this ticket? Should this have been removed?

Reply:
No, Added the feature flag again.